File: answers/Fab-Gaurav/Day 2/Answer_2.py
#Program to check if the number "n" is an Disarium Number

n=int(input("Enter the number : "))
j=1
sum=0
temp=n
while(temp!=0):
    rem=temp%10
    sum=sum*10+rem
    temp=int(temp/10)
temp=sum
sum=0
while(temp!=0):
    rem=temp%10
    sum=sum+pow(rem,j)
    temp=int(temp/10)
    j=j+1
if sum==n:
    print("%d is a Disarium Number"%n)
else:
    print("%d is not a Disarium Number"%n)

File: answers/Shobhit403/DAY 10/Answer2.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
int n=0;
string c;
cout<<"Enter the string to check: ";
cin>>c;
for (int i = 0; i < c.size(); i++) 
{
    if(c[i]>=65 && c[i]<=91)
    {
        cout<<endl;
        n++;
    }
    cout<<c[i];
}
 cout<<endl<<n;
return 0;    
}
